### PR TITLE
Repository.List: Add nebula-preview to receive visibility field

### PR DESCRIPTION
**Pull request by andygrunwald**
**What type of PR is this?**

feature

**What this PR does / why we need it**:

When calling `Repositories.List` (API Method: [List repositories for the authenticated user](https://docs.github.com/en/rest/reference/repos#list-repositories-for-the-authenticated-user)), the field `visibility` is always `nil`.

The `visibility` was introduced in a developer / API preview controlled by the media accept header. 
This PR enabled the API preview `nebula-preview` for `Repositories.List`, to retrieve the `visibility` of a repository.

**Special notes for your reviewer**:

None

**Does this PR introduce a user-facing change?**:

None

**Additional documentation e.g., usage docs, etc.**:

- [List repositories for the authenticated user @ GitHub REST API docs](https://docs.github.com/en/rest/reference/repos#list-repositories-for-the-authenticated-user): In the response, the `visibility` field is available.
- [API changes to support internal repository visibility (2019-12-03)](https://developer.github.com/changes/2019-12-03-internal-visibility-changes/#repository-visibility-fields)

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @andygrunwald !
LGTM.

Awaiting second LGTM before merging.

**Review by wesleimp — APPROVED** [review]
LGTM 👌🏼

**Comment by gmlewis** [comment]
Thank you, @wesleimp !
Merging.

**Pull request merged**
